scripts/1strequest.sh

```diff
@@ -5,13 +5,13 @@
 
 # Example usage
 # 1) Run the Spring app 10 times
-# $ ./1strequest.sh "java -XX:ActiveProcessorCount=8 -Xms512m -Xmx512m -jar ../springboot3/target/app.jar" 10
+# $ ./1strequest.sh "java -XX:ActiveProcessorCount=8 -Xms512m -Xmx512m -jar ../springboot3/target/springboot3.jar" 10
 #
 # 2) Run the Quarkus app 10 times
-# $ ./1strequest.sh "java -XX:ActiveProcessorCount=8 -Xms512m -Xmx512m -jar ../quarkus3/target/quarkus-app/quarkus-run.jar" 10
+# $ ./1strequest.sh "java -XX:ActiveProcessorCount=8 -Xms512m -Xmx512m -jar ../quarkus3/target/quarkus-app/quarkus3.jar" 10
```

Comment:
I don't think this is correct?

```bash
╭─ ~/w/quarkus/benchmarking/spring-quarkus-perf-comparison-upstream/quarkus3  main ······································· ✔  4s 
╰─ ls -l target/quarkus-app
total 32
drwxr-xr-x@ 3 edeandre  staff    96 Oct 16 18:22 app
drwxr-xr-x@ 4 edeandre  staff   128 Oct 16 18:22 lib
drwxr-xr-x@ 5 edeandre  staff   160 Oct 16 18:22 quarkus
-rw-r--r--@ 1 edeandre  staff  8541 Oct 16 18:22 quarkus-app-dependencies.txt
-rw-r--r--@ 1 edeandre  staff   646 Oct 16 18:22 quarkus-run.jar
```

Reply:
Weird. This is what I see after `mvn clean` and `mvn install`. I just rebased and it's the same:

```
holly@hcummins-mac spring-quarkus-perf-comparison % ls */target/*jar
quarkus3-spring-compatibility/target/app.jar    quarkus3/target/quarkus3.jar                    springboot3/target/springboot3.jar
```